### ISSUE TITLE
Syntax highlighting CSS

### ISSUE THREAD

**Issue by tripflex** [mAAdhaTTah/WP-Gistpen#62]
With current release on WP repo I had to manually edit the CSS files and add `!important` to prevent my theme's CSS from being used on any syntax highlighting.  Not sure if this is something that could be done easily or not ... but any of the syntax highlighting CSS "technically" should be prioritized over theme CSS 

... i'll try and see how it goes with develop branch, but thought I would mention it as i'm sure others who try the plugin out and have the same issue, probably just give up on it instead of digging further into how to fix it themselves (even if it's the theme's basic CSS causing problems)

**Comment by mAAdhaTTah** [comment]
Avoid the `develop` branch for now, please. It's still pretty alpha, and I can't guarantee anything at this point.

This is really more of an issue with PrismJS, tbh. The classes on its CSS is pretty thinly nested, and uses some generic class names. If you're writing your own CSS, I'd suggest changing those classes not to conflict with PrismJS, if possible. It's an issue that [comes up](https://github.com/PrismJS/prism/issues/1044) from [time to time](https://github.com/PrismJS/prism/issues/6) on that repo, but it doesn't seem like they're going to fix that.

That said, I could do something in the plugin to resolve it, add an extra class nesting with SASS pretty easily. I'll keep this open to take a look into it.

**Comment by tripflex** [comment]
@mAAdhaTTah any word on this?  TBH i'm still using this old modified version and was lurking around thinking about upgrading but didn't want to have to spend hours messing with CSS again lol

If not no worries, once I switch themes I can come back and revisit this -- but for now i'm using a theme built by someone else and haven't finished my custom one yet

**Comment by tripflex** [comment]
Ahh just noticed the upgrade notice in changelog to 1.x.x -- so yeah i'll just wait until i finish up new site and then just do clean install of latest :)

**Comment by mAAdhaTTah** [comment]
Yeah, we're well into the 1.x release, so feel free to go with that version now :).

There is a prefix plugin for Prism, but it does require updating / modifying the themes, so that's the main issue. Because this is more of an upstream issue, I'm hesitant to fix it for WP-Gistpen, and maintaining forks of all the themes with custom prefixes would be pretty time-intensive.
